kvmagent/kvmagent/plugins/nvram/nvram_local.py

```
@@ -62,11 +63,11 @@ def _check_raw_has_file_system(self, nvram_install_path):
         is_fs = info.get('USAGE') == 'filesystem'
         return is_ext4 and is_label_ok and is_fs
 
-    def _make_ext4_nvram_filesystem(self, nvram_install_path):
-        cmd = "mkfs.ext4 -F -L VM_NVRAM '%s'" % nvram_install_path
+    def _make_ext4_nvram_filesystem(self):
+        cmd = "mkfs.ext4 -F -L VM_NVRAM '%s'" % self.nvram_install_path
         ret = bash.bash_r(cmd) # type: str
         if ret != 0:
-            raise Exception("Failed to format ext4 on %s" % nvram_install_path)
+            raise Exception("Failed to format ext4 on %s" % self.nvram_install_path)
```

Ruff (0.15.2)

[warning] Create your own exception

(TRY002)